.github/workflows/conformance-ipsec-e2e.yaml

```diff
@@ -349,14 +358,14 @@ jobs:
           key-type-two: ${{ matrix.key-type-two }}
           encryption-overlay: ${{ matrix.encryption-overlay }}
 
+      - name: Assert that no unencrypted packets are leaked
+        uses: ./.github/actions/bpftrace/check
+
```

Comment:
Is it intentional that the second connectivity suite run is no longer checked for leaks?

Reply:
Not really, I just brought it back.